src/mapping.js

```diff
@@ -26,16 +26,27 @@ class Mapping {
 	}
 
 	walk(root, func) {
+		//IE11 has strange issue, if root is text node IE throws exception on
+        //calling treeWalker.nextNode(), saying
+        //Unexpected call to method or property access instead of returing null value
+        if(root && root.nodeType === Node.TEXT_NODE) {
+        	return;
```

Comment:
IE 11 it throw exception if we don't have this condition. in other browsers like chrome it `treeWalker.nextNode()` is generated as null and it skips the `func(node)`.